Makefile

```diff
@@ -2,10 +2,18 @@ PROG=	bsdec2-image-upload
 SRCS=	main.c
 MAN	=
 WARNS	?=	3
-CFLAGS	+=	-Wno-error=\#warnings
 BINDIR	?=	/usr/local/bin
 LDADD	+=	-lcrypto -lssl
 
+UNAME_S := $(shell uname -s)
+.if (${UNAME_S},Linux)
+CERTFILE = /etc/ssl/certs/ca-bundle.crt
+CFLAGS += --no-warnings
```

Comment:
Umm... no.

Reply:
How would you suggest defining CERTFILE in the makefile?